app/report/Report.tsx

```diff
@@ -0,0 +1,66 @@
+"use client";
+import { useState } from "react";
+import { useMemo } from "react";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import classNames from "classnames";
+import { ReportResidencyFrTaxesFr } from "./residency_fr_taxes_fr/ReportResidencyFrTaxesFr";
+
+export type CountryCode = "us" | "fr";
+export interface ReportProps {}
+
+export const Report: React.FunctionComponent<ReportProps> = () => {
+  const [taxesCountry, setTaxesCountry] = useState<Partial<CountryCode>>("fr");
+  const [residency, setResidencyCountry] = useState<Partial<CountryCode>>("fr");
+
+  const queryClient = useMemo(() => new QueryClient(), []);
+
+  return (
+    <QueryClientProvider client={queryClient}>
```

Comment:
should we share a single `QueryClientProvider` across the app?

Reply:
Done in https://github.com/hinosxz/tax-helper/pull/23